CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # edge-react-gui
 
+## 2.13.0 (2022-04-11)
```

Comment:
Should also mention that now the app can build natively on M1 macs without rosetta.